dm_cli/import_entity.py

```diff
@@ -96,16 +103,14 @@ def import_folder_entity(
     destination_path = Path(destination)
     data_source = destination_path.parts[0]
 
-    try:  # Check if target already exists on remote. Then delete or raise exception
-        dmss_api.document_get(f"dmss://{destination}/{source_path.name}")
+    # Check if target already exists on remote. Then delete or raise exception
+    target = f"{destination}/{source_path.name}"
+    exists = dmss_api.document_check(target)
+    if exists:
         if not state.force:
-            raise ValueError(f"Failed to upload to 'dmss://{destination}/{source_path.name}' - It already exists.")
-        console.print(
-            f"WARNING: '{destination}/{source_path.name}' already exists. Replacing it...", style="dark_orange"
-        )
-        dmss_api.document_remove(f"{destination}/{source_path.name}")
-    except NotFoundException:
-        pass  # The folder we're trying to upload does not exist, which is fine
+            raise ValueError(f"Failed to upload to '{target}' - It already exists.")
+        console.print(f"'{target}' already exists.  Replacing it...", style="dark_orange")
+        remove_by_path_ignore_404(target)
```

Comment:
How can this be right?
If we ignore 404 then the target will not be deleted. And we end up with duplicates

Reply:
If delete returns a 404 it means that there is nothing to delete there, right?